airflow/www_rbac/templates/appbuilder/baselayout.html

```diff
@@ -70,6 +70,13 @@
   // below variables are used in clock.js
   var hostName = '{{ hostname }}';
   var csrfToken = '{{ csrf_token() }}';
+  $.ajaxSetup({
+    beforeSend: function(xhr, settings) {
+      if (!/^(GET|HEAD|OPTIONS|TRACE)$/i.test(settings.type) && !this.crossDomain) {
+        xhr.setRequestHeader("X-CSRFToken", "{{ csrf_token() }}");
```

Comment:
@gsilk can you please use the `csrfToken` variable that is declared above instead of  calling the `csrf_token()` again?